comeet/UI/MyAgenda/MyAgendaViewController.swift

```diff
@@ -77,8 +76,8 @@ private extension MyAgendaViewController {
     }
     
     func goToSelectedSection() {
```

Comment:
Actually, I'm pretty sure this is incorrect.  The bug isn't around scrolling to the right spot in the agenda screen (though that's needed too), it's about fetching the appointments to display from the right date.